recipes/cuda-nvml-dev/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "cuda-nvml-dev" %}
+{% set version = "12.0.76" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_nvml_dev/{{ platform }}/cuda_nvml_dev-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 6780e554784bef73050144afe39668e26923d2533c268460c348e0e380d5d048  # [linux64]
+  sha256: 319b96999d144192094d254f0607debb14118ce206ef7dfeba3d13da1cc805ec  # [ppc64le]
+  sha256: 4aee09e48ecd30b2dcaa9de4eb8af2fcdf36ed040eac50c14f3345a12410b6ec  # [aarch64]
+  sha256: abc366280d0aa1ca047323a3e316270b80ffabb649021184647cd16b44c60a71  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
```

Comment:
Most of the other packages have listed a sysroot here. Should we copy that? Is it needed?

Reply:
The default is to build on CentOS 6 with GLIBC 2.12. If that works, then we don't need `sysroot`

If this requires CentOS 7 with GLIBC 2.17, then we need to add `sysroot` (as done elsewhere)

Reply:
Okay. So if CI passes without it, we don't need it. Sounds good.